plugins/standard_tests/newhelm/tests/simple_safety_tests.py

```diff
@@ -0,0 +1,129 @@
+import csv
+from typing import List, Dict, Mapping
+
+from newhelm.aggregations import mean_of_measurement
+from newhelm.annotators.llama_guard_annotator import (
+    LlamaGuardAnnotation,
+    LlamaGuardAnnotator,
+)
+from newhelm.base_annotator import BaseAnnotator
+from newhelm.base_test import BasePromptResponseTest, TestMetadata
+from newhelm.dependency_helper import DependencyHelper
+from newhelm.external_data import ExternalData
+from newhelm.placeholders import Prompt, Result, SUTOptions
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    PromptWithContext,
+    TestItemAnnotations,
+    TestItem,
+)
+from newhelm.test_registry import TESTS
+
+
+_HARM_AREAS = [
+    "Illegal and Highly Regulated items",
+    "Physical harm and Violence",
+    "Scams and fraud",
+    "Child safety",
+    "Suicide, Self-Harm, and Eating Disorders",
+]
+
+# TODO: Other languages?
+
+
+class SimpleSafetyTests(BasePromptResponseTest):
+    """
+    SimpleSafetyTests consists of 100 prompts over five severe harm-areas for the rapid identification of critical safety risk in an LLM.
+    The harm-areas are as follows:
+        - Illegal Items
+        - Physical Harm
+        - Scams & Farud
+        - Child Abuse
+        - Suide, Self-Harm & Eating Disorders.
+    The test suite consists of two different types of prompts:
+        - Information/Advice Seeking
+        - Insutrctions/Actions
+
+    It is important to note the negative predictive power of this test-- it is assessing the unsafety of a SUT, rather than the safety. Passing this test only indicates that no critical safety risks have been found; it does not guarantee that no such risks exist.
+    """
+
+    @classmethod
+    def get_metadata(cls) -> TestMetadata:
+        return TestMetadata(
+            name="SimpleSafetyTests",
+            description="Idenitfy crtical safety risks for chat LLMs. ",
+        )
+
+    @classmethod
+    def get_dependencies(cls) -> Mapping[str, ExternalData]:
+        # Test data is included in newhelm repository.
+        return {}
+
+    def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]:
+        sut_options = SUTOptions(
+            temperature=0.01,
+            num_completions=1,
+            max_tokens=512,  # TODO: Change to 1048. HuggingFace .generate() raises exception when this is > than model's pre-defined max length (GPT2.n_positions = 1024)
+        )
+        test_items: List[TestItem] = []
+        data_dir = "plugins/standard_tests/newhelm/tests/test_data/SimpleSafetyTests.csv"  # This probably needs to be changed.
```

Comment:
IMO data files shouldn't go into NewHELM, even if they are small. One of the goals we outlined in [this doc](https://docs.google.com/document/d/1HcgSk4v6v4dJtVek62t1K8ErpxQB4dOy09mKuLzI8ec/edit#heading=h.fni9lnrwpxlx) was knowing the data's original source. This data didn't originate in our repo. It also likely won't stay up to date with the author's desires.

The blessed path for this would be to create a new ExternalData subclass, something like `GoogleDriveData`. It looks like Old HELM has some [logic for how to do the actual download](https://github.com/stanford-crfm/helm/blob/633c47d5321f9ebd5a3382e577d9ad1d03402cb9/src/helm/common/general.py#L84-L92).

Reply:
That makes sense and I can do that, but I don't think the AI Safety/ Tests & Benchmarks google drive is public.

Reply:
I agree the data shouldn't go into NewHELM, but isn't this part of what a plugin mechanism is good for? Test authors can just throw their data in their git repo, publish official artifacts containing it, and then we have a reliable source for both code and data.